content/agent/_index.md

```diff
@@ -3,103 +3,137 @@ title: Agent
+### Get more logging from the agent
 
-### Issues getting the Agent reporting
+To enable the full debug mode:
 
-If you get the Agent installed but are not seeing any data in Datadog, you can troubleshoot in the following manner.
-First, [run the info command](/agent/faq/agent-status-and-information). Select your OS in the navigation column on the left of this page to see how to run this. Does running the [info command](/agent/faq/agent-status-and-information) show any errors?
+1. Modify your local `datadog.yaml` file (see [this page](/agent/v6/#configuration-file) to locate this configuration file on your instance)
 
-If not, you should also check the logs (location of the logs again depends on OS). Errors in the logs may also reveal the cause of any issues.
+2. Replace `# log_level: INFO` with `log_level: DEBUG` (make sure to get rid of # to uncomment the line)
 
-If not, send both the full output of the [info command](/agent/faq/agent-status-and-information) and the logs with a flare as attachments to the [Datadog support team](mailto:support@datadoghq.com?Subject=Agent%20issues).
+3. Restart your Datadog Agent (see [that page](/agent/#start-stop-restart-the-agent) to find the restart command depending on your OS)
 
-#### Check your machine's time
-We have also seen a few cases where machines have their clock set further in the future or the past, which can sometimes cause problems with metric submission.
-To check for this, run:
-
-```shell
-date -u && curl -s -v https://app.datadoghq.com 2>&1 | grep Date
-```
-This outputs the current system’s date, and then makes a request to our endpoint and grabs the date on our end.
-If these are more than a few minutes apart, you should correct the time settings on your server.
+4. Wait a few minutes to generate some logs. [Look here](/agent/#log-locations) to find the location of the logs.
 
-### Issues getting integrations working
+### Send a flare
 
-Datadog has many [integrations](/integrations/) which are set up through [YAML files in the Agent](https://github.com/DataDog/dd-agent/tree/master/conf.d).
 
-Here is a quick guide for troubleshooting [integrations](/integrations) installation:
+If you are running the 5.3 version (or higher) of the agent, you're able to send all necessary troubleshooting information to our Support Team, with one flare command!
 
-1. [Run the info command](/agent/faq/agent-status-and-information).
+`flare` gathers all of the agent's configuration files and logs into an archive file. It removes sensitive information including passwords, API keys, Proxy credentials, and SNMP community strings.  
+**Confirm the upload of the archive to immediately send it to Datadog support**.  
+Since the Datadog Agent is completely open source, you can [verify the code's behavior](https://github.com/DataDog/dd-agent/blob/master/utils/flare.py). You can also review the archive prior to sending as the flare prompts a confirmation before uploading it.  
```

Comment:
inconsistent capitalization again.

also we seem to be linking to v5 source code?

"Since the Datadog Agent is open source, you can audit the code
 for the [flare command](https://github.com/DataDog/dd-agent/blob/master/utils/flare.py). You can also review the archive prior to sending as the flare prompts a confirmation before uploading it"

Reply:
yup, I have to update all link pointing to dd-agent.. But i had plan to do it at the end of this PR review.